docs/src/ActivationCard.doc.js

```diff
@@ -0,0 +1,162 @@
+        required: false,
+        defaultValue: null,
+        description: [
+          'Callback fired when the dismiss button is clicked (pressed and released) with a mouse or keyboard.',
+          'Supply a short, descriptive label for screen-readers to provide sufficient context about the dismiss button action. IconButtons do not render text for screen readers to read requiring an accessibility label.',
+          'Accessibility: `accessibilityLabel` populates aria-label.',
+        ],
+        href: '',
+      },
+      {
+        name: 'link',
+        type:
+          '{| accessibilityLabel?: string , href: string, label: string, onClick?: ({ event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement | SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement> }) => void |}',
+        required: false,
+        defaultValue: null,
+        description: [
+          'Link-role button to render inside the activation card as a call-to-action to the user.',
+          '- label: Text to render inside the button to convey the function and purpose of the button. The button text has a fixed size.',
+          '- accessibilityLabel: Supply a short, descriptive label for screen-readers to replace button texts that do not provide sufficient context about the button component behavior. Texts like `Click Here,` `Follow,` or `Read More` can be confusing when a screen reader reads them out of context. In those cases, we must pass an alternative text to replace the button text.',
+          '- onClick: Callback fired when the button component is clicked (pressed and released) with a mouse or keyboard.',
+          'Accessibility: `accessibilityLabel` populates aria-label. Screen readers read the `accessibilityLabel` prop, if present, instead of the button `text`.',
+        ],
+        href: '',
+      },
+      {
+        name: 'status',
+        type: `"notStarted" | "pending" | "needsAttention" | "complete"`,
+        required: true,
+        defaultValue: null,
+        description: [
+          'Select the activation card status:',
+          '-`notStarted`: A task that has not be started',
+          '-`pending`: A task that is pending action',
+          "-`needsAttention`: A task that requires the user's attention",
+          '-`complete`: A task that has been completed',
+        ],
+        href: '',
+      },
+      {
+        name: 'statusMessage',
+        type: 'string',
+        required: true,
+        defaultValue: null,
+        description: [
+          'A message to indicate the current status of the activation card.',
+        ],
+        href: '',
+      },
+      {
+        name: 'title',
+        type: 'string',
+        required: true,
+        defaultValue: null,
+        description: [
+          'Heading to render inside the activation card above the message to convey the activation card topic to the user. The message text has a fixed size.',
```

Comment:
@norrise120 Can we remove `The message text has a fixed size` (looks like that should be for the `message` prop)